src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,25 +29,30 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${todo?.id}`}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-            <button type="button" className="delete" data-cy="modal-close" />
+            <button
+              type="button"
+              className="delete"
+              data-cy="modal-close"
+              onClick={() => onShow(false)}
+            />
           </header>
 
           <div className="modal-card-body">
             <p className="block" data-cy="modal-title">
-              quis ut nam facilis et officia qui
+              {todo?.title}
             </p>
 
             <p className="block" data-cy="modal-user">
               {/* <strong className="has-text-success">Done</strong> */}
-              <strong className="has-text-danger">Planned</strong>
+              <strong className="has-text-danger">{`${todo?.completed ? 'Done' : 'Planned'}`}</strong>
```

Comment:
- do not keep commented code
- should be deferent classes for completed and not completed
- do not complecate ternary
```suggestion
              <strong className="has-text-danger">{todo?.completed ? 'Done' : 'Planned'}</strong>
```